### PR TITLE
add script for google events, fill in event data for CED

### PR DESCRIPTION
**Pull request by shughes26**
This will also allow us to fill out `event_data` fields for future workshops to show up in google events

https://developers.google.com/search/docs/advanced/structured-data/event

### CODE REVIEW

File: themes/default/layouts/robots.txt [hunk 1]
```diff
@@ -3,6 +3,10 @@ User-agent: *
 Disallow: *
 {{ end }}
 
+# allow Googlebot for events
+User-agent: Googlebot
+Disallow:
+
```
Comment (shughes26) [inline themes/default/layouts/robots.txt:9]: If someone knows about how this stuff works please let me know!

File: themes/default/layouts/partials/head.html [hunk 2]
```diff
@@ -27,6 +27,32 @@
...
+                "@context": "https://schema.org",
+                "@type": "Event",
+                "name": "{{ .Params.event_data.name }}",
+                "startDate": "{{ .Params.event_data.start_date }}",
+                "endDate": "{{ .Params.event_data.end_date }}",
+                "eventStatus": "https://schema.org/EventScheduled",
+                "eventAttendanceMode": "https://schema.org/OnlineEventAttendanceMode",
+                "location": {
+                  "@type": "VirtualLocation",
+                  "url": {{ .Params.event_data.url }}
```
Comment (zchase) [inline themes/default/layouts/partials/head.html:43]: Does this need quotes?
```suggestion
                  "url": "{{ .Params.event_data.url }}"
```

Comment (shughes26) [reply themes/default/layouts/partials/head.html:43]: Weirdly, no. Adding quotes escapes the URL, JSON comes out clean without them.

Comment (shughes26) [reply themes/default/layouts/partials/head.html:43]: (I was absolutely baffled by this earlier)

**Review by zchase — APPROVED** [review]

**Pull request merged**